hawkular-wildfly-agent-feature-pack/src/main/resources/configuration/standalone/template.xml

```diff
@@ -18,4 +18,85 @@
 
 -->
 <server xmlns="urn:jboss:domain:4.0">
+
+  <extensions>
+        <?EXTENSIONS?>
+  </extensions>
+
+  <management>
```

Comment:
Currently they don’t.  We read them as humans to translate to our Java API to configure it.  So they’re useful to me, as a person.

That said, @heiko-braun is doing work to let -swarm play with standalone.xml and such, so we may soon reach the point where we are consuming or otherwise inspecting the XML fragments you guys ship in the feature-packs.

-Bob


> On Jan 14, 2016, at 4:13 PM, Peter Palaga <notifications@github.com> wrote:
> 
> In hawkular-wildfly-agent-feature-pack/src/main/resources/configuration/standalone/template.xml <https://github.com/hawkular/hawkular-agent/pull/137#discussion_r49785973>:
> 
> > @@ -18,4 +18,85 @@
> >  
> >  -->
> >  <server xmlns="urn:jboss:domain:4.0">
> > +
> > +  <extensions>
> > +        <?EXTENSIONS?>
> > +  </extensions>
> > +
> > +  <management>
> @bobmcwhirter <https://github.com/bobmcwhirter> do these subsystems.xml and template.xml files have any significance for WF Swarm?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/hawkular/hawkular-agent/pull/137/files#r49785973>.
>